### PR TITLE
testing project

### PR DESCRIPTION
**Pull request by jaspreet-singh-sahota**
# This project is an introduction to the use of Jest.
- Used Jest for testing js functions

### CODE REVIEW

File: src/calculator.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  multiply() {
+    return this.num1 * this.num2;
+  }
+
+  divide() {
+    return this.num1 / this.num2;
+  }
+}
```
Comment (Oghenebrume50) [inline src/calculator.js:22]: ideally calculator should have methods that control their own behaviour, in the sense that, add, subtract,.... should be able to take numbers themselves and manipulate them, currently you are passing the numbers to calculator and calling the operation on it

File: src/ceaser_cipher.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const cipherWord = (word, num) => word.split('').map(char => (char === char.toUpperCase()
+  ? String.fromCharCode(65 + ((char.charCodeAt() - 65 + num) % 26))
+  : String.fromCharCode(97 + ((char.charCodeAt() - 97 + num) % 26)))).join('');
+
+const cipher = (str, num) => {
+  const arr = str.split(' ');
+  return arr.map(word => cipherWord(word, num)).join(' ');
+};
+
+export default cipher;
```
Comment (Oghenebrume50) [inline src/ceaser_cipher.js]: Special characters or punctuations should be preserved and not , encrypted
see this

File: test/ceaser_cipher.test.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import cipher from '../src/ceaser_cipher';
```
Comment (Oghenebrume50) [inline test/ceaser_cipher.test.js:1]: Need to test that punctuations are preserved :slightly_smiling_face:

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @jaspreet-singh-sahota @frankopkusianwar  good job team :+1: this was well implemented. 

However, I have highlighted some areas that needs improvement, please check those out make changes 
and request for another review :slightly_smiling_face:  

**RAPHAEL NORIODE** :mask: stay_safe

File: src/calculator.js (hunk 1 above)
Comment (jaspreet-singh-sahota) [reply src/calculator.js:22]: Ideally, a problem can be solved in many ways there is no one particular way of solving an algorithm. your suggestion is good, but, I prefer class over functions any day.

Jaspreet Singh   (You too stay safe)

**Review by shloch — APPROVED** [review]
## Hello @jaspreet-singh-sahota , @frankopkusianwar 

Good to go..great job

Screen Shot 2020-07-28 at 10 04 39 PM

Can merge this

**Pull request merged**